pkg/function/scale_workload.go

```diff
@@ -59,11 +60,19 @@ func (*scaleWorkloadFunc) RequiredArgs() []string {
 }
 
 func getArgs(tp param.TemplateParams, args map[string]interface{}) (namespace, kind, name string, replicas int32, err error) {
-	err = Arg(args, ScaleWorkloadReplicas, &replicas)
+	var rep interface{}
+	err = Arg(args, ScaleWorkloadReplicas, &rep)
 	if err != nil {
 		return namespace, kind, name, replicas, err
 	}
-
+	if val, ok := rep.(int); ok {
+		replicas = int32(val)
+	} else if val, ok := rep.(string); ok {
+		strToInt := intstr.Parse(val)
+		replicas = strToInt.IntVal
+	} else {
+		return namespace, kind, name, replicas, errors.Wrapf(err, "Failed to decode arg `%s`", ScaleWorkloadReplicas)
```

Comment:
typically I'd use a naked return with named return values. 
```suggestion
		err = errors.Errorf("Invalid arg type %T for Arg %s ", rep, ScaleWorkloadReplicas)
		return
```